templates/zerver/compose.html

```diff
@@ -78,6 +79,9 @@
                      id="attach_files" href="#" title="{{ _('Attach files') }}"></a>
                   <a class="message-control-button icon-vector-font"
                      href="#markdown-help" title="Formatting" data-toggle="modal"></a>
+                  <a class="markdown_preview_button" style="display:none;"
```

Comment:
I'm sorry, I still find this confusing.  You now have a tag whose class is markdown_preview_button but its id is undo_markdown_preview.  Somebody scanning the code is going to think this is the button that turns on preview when they read the class, which comes first.  I know the class refers to overall feature, but I'm not sure you really need it.  In the CSS where the two buttons have characteristics in common, you can just use both id selectors separated by a comma.